docs/api/global-shortcut.md

```diff
@@ -47,6 +47,8 @@ The `globalShortcut` module has the following methods:
 * `accelerator` [Accelerator](accelerator.md)
 * `callback` Function
 
+Returns `Boolean` - Whether or not the shortcut was registered successfully.
```

Comment:
This change does not introduce a new return value: it documents a return value that was previously undocumented